### PR TITLE
chore: prepare release v0.96.0-sumo-0

### PR DESCRIPTION
**Pull request by kkujawa-sumo**

### CODE REVIEW

**Comment by rnishtala-sumo** [comment]
Looks like there's an issue with this link in the upgrading doc.

- https://github.com/SumoLogic/opentelemetry-collector-contrib/tree/v0.54.0-filterprocessor/processor/filterprocessor → Status: 404

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+- feat(sumologicexporter)!: remove deprecated json_logs [#1452]
+- feat(sumologicexporter)!: remove deprecated clear_logs_timestamp [#1455]
+
+### Added
+
+- ci: build ubi-based images [#1440]
+- feat(k8sprocessor): extract namespace annotations [#1457]
+- ci: build UBI-based FIPS images [#1463]
+- feat: build windows containers [#1467]
```
Comment (swiatekm-sumo) [inline CHANGELOG.md:24]: I think we're only building these on `main`, so this in the changelog might confuse users. WDYT @sumo-drosiek ?

Comment (astencel-sumo) [reply CHANGELOG.md:24]: If a feature is not available in the release builds, we should either not mention it at all in the README, or explicitly call out that (like in this case) it is only available for development builds.

**Comment by kkujawa-sumo** [comment]
> * https://github.com/SumoLogic/opentelemetry-collector-contrib/tree/v0.54.0-filterprocessor/processor/filterprocessor

I'll change the link to https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.54.0/processor/filterprocessor

File: CHANGELOG.md (hunk 1 above)
Comment (kkujawa-sumo) [reply CHANGELOG.md]: I change this changelog enty, I hope that now it is clear

**Comment by swiatekm-sumo** [comment]
If we're going to release this, can we add the stanza recombine issue to known issues and point it out in the release notes?

**Comment by kkujawa-sumo** [comment]
> If we're going to release this, can we add the stanza recombine issue to known issues and point it out in the release notes?

@swiatekm-sumo I added this here: https://github.com/SumoLogic/sumologic-otel-collector/pull/1479/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR14-R20
do you think about different form?

**Review by swiatekm-sumo — APPROVED** [review]

**Review by astencel-sumo — APPROVED** [review]

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**